uber/configspec.ini

```diff
@@ -419,7 +419,9 @@ dealer_reg_public      = string(default="%(dealer_reg_start)s")
 supporter_deadline = string(default="")
 
 # The date at which donation tiers at SHIRT_LEVEL and higher but below
-# SUPPORTER_LEVEL can no longer be selected by attendees.
+# SUPPORTER_LEVEL can no longer be selected by attendees.  We also use this as
+# the deadline after which staffers may no longer toggle whether they want an
+# event shirt or multiple staff shirts.
 shirt_deadline = string(default="")
```

Comment:
I originally added a new config option for this new deadline, but I eventually realized that it made the most sense to just use this one.  I checked and confirmed that it's set for Dec 1, which is when we promised to send over this info anyway.

Technically this does put us in the position of sending over the info at literally the last minute of the day, and if we decide that we want to set a different, earlier deadline for the "two staff shirts vs 1 of each" question then we'll need a new config option.